src/assisted_test_infra/test_infra/controllers/node_controllers/nutanix_controller.py

```diff
@@ -66,6 +66,10 @@ def get_ingress_and_api_vips(self):
                 "ingress_vips": self._entity_config.ingress_vips,
             }
 
+        elif self._entity_config.vip_dhcp_allocation is True:
```

Comment:
```suggestion
        else:
```

should be enough?

Reply:
Nope, we need to differ True, from False from None.
`else` in this case will include the `None` case

Reply:
can we add some doc at the beginning of te function to describe the 3 cases? I think it would help to understand what we try to achieve

Reply:
There is https://github.com/openshift/assisted-test-infra/pull/2423/files#diff-6016d1ff333f74b2b2703cf120015d2daea4cd905295107122b40f81a7b7e435R72

Reply:
I can elaborate more if needed

Reply:
yeah, it's just because I find this structure a bit misleading to read:
```
def f(param):
  if param==true:
    return
  if param==false
    return
  
  ... more code here because param is none...

```

Reply:
@adriengentil I added a function documentation, LMK if that is OK

Reply:
perfect, thanks! Just a typo.